src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -99,6 +99,15 @@ private enum Mode
         }
 
         /// <summary>
+        /// Address of the linked resource.
+        /// </summary>
+        /// <remarks>
+        /// Passed through to the generated HTML in all cases.
+        /// </remarks>
+        [HtmlAttributeName(HrefAttributeName)]
+        public string Href { get; set; }
```

Comment:
nah.  not much more to say and existing "Passed through" comment matches what we have other places we bind a regular HTML attribute.